File: Javascript/A_NivelIniciante/Exercicio_1019.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var segundos = parseInt(lines.shift())
var h = segundos/3600
console.log(parseInt(h)+ ":" + parseInt((h - parseInt(h))*60) + ":" + parseInt(segundos%60))

File: Python/A_NivelIniciante/Exercicio_1019.py
# -*- coding: utf-8 -*-

N = int(input())
horas = N/3600
minutos = (N - (horas * 3600))/60 
segundos = N -((horas * 3600) + (minutos * 60))
print("%s:%s:%s" %(str(horas),str(minutos),str(segundos)))